src/optimisation/Optimisation.jl

```diff
@@ -1,324 +1,327 @@
 module Optimisation
 
 using ..Turing
+using ..OptimisationCore: ModeResult, MLE, MAP, OptimLogDensity, OptimizationContext,
+    ModeEstimator, variable_names
+using NamedArrays: NamedArray
+using Optimization
+using OptimizationOptimJL: LBFGS, IPNewton
+using DocStringExtensions: TYPEDFIELDS
 using Bijectors
 using Random
-using SciMLBase: OptimizationFunction, OptimizationProblem, AbstractADType, NoAD
+using SciMLBase: OptimizationFunction, OptimizationProblem, solve
```

Comment:
This is strange: how is the current code working at all o.O

When you do `using SciMLBase: OptimizationProblem`, you can't overload it. So the re-definition below should cause issues AFAIK o.O

But this will be addressed when using qualifying references, as outlined in the comment below.

Reply:
Good point, don't know what was going on there. Now using `import`.